File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/domain/converter/TbspToCupConverter.java
package com.tarbadev.witchcraft.domain.converter;

public class TbspToCupConverter implements IConverter {
  private static final Double TBSP_IN_ONE_CUP = 16.0;

  @Override
  public Double convert(Double number) {
    return number / TBSP_IN_ONE_CUP;
  }
}

File: WitchcraftApp/src/main/java/com/tarbadev/witchcraft/converter/TbspToCupConverter.kt
package com.tarbadev.witchcraft.converter

class TbspToCupConverter : IConverter {

    override fun convert(number: Double): Double {
        return number / TBSP_IN_ONE_CUP
    }

    companion object {
        private const val TBSP_IN_ONE_CUP = 16.0
    }
}
